slack/web/classes/objects.py

```diff
@@ -344,7 +344,7 @@ def __init__(
         *,
         value: str,
         label: Optional[str] = None,
-        text: Optional[Union[str, dict, TextObject]] = None,  # Block Kit
+        text: Optional[Union[str, dict, PlainTextObject]] = None,  # Block Kit
```

Comment:
As mentioned here - this `text` can be `mrkdwn` type. Can you revert this change? https://api.slack.com/reference/block-kit/composition-objects#option

>Overflow, select, and multi-select menus can only use plain_text objects, while radio buttons and checkboxes can use mrkdwn text objects. Maximum length for the text in this field is 75 characters.